.readthedocs.yml

```diff
@@ -0,0 +1,22 @@
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: all
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.7
+  install:
+    - requirements: docs/requirements.txt
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
```

Comment:
have we tried to reproduce smth [like that](https://github.com/readthedocs/readthedocs.org/issues/4912#issuecomment-483756581)?